pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -117,3 +127,29 @@ func (brokerRESTStrategy) ValidateUpdate(ctx kapi.Context, new, old runtime.Obje
 
 	return scv.ValidateBrokerUpdate(newBroker, oldBroker)
 }
+
+func (brokerStatusRESTStrategy) PrepareForUpdate(ctx kapi.Context, new, old runtime.Object) {
+	newBroker, ok := new.(*sc.Broker)
+	if !ok {
+		glog.Warning("received a non-broker object to update to")
+	}
+	oldBroker := old.(*sc.Broker)
+	if !ok {
+		glog.Warning("received a non-broker object to update from")
+	}
+	// status changes are not allowed to update spec
+	newBroker.Spec = oldBroker.Spec
+}
+
+func (brokerStatusRESTStrategy) ValidateUpdate(ctx kapi.Context, new, old runtime.Object) field.ErrorList {
+	newBroker, ok := new.(*sc.Broker)
+	if !ok {
+		glog.Warning("received a non-broker object to validate to")
+	}
+	oldBroker := old.(*sc.Broker)
+	if !ok {
```

Comment:
here we actually have an ErrorList to use. I'm not sure if it's specifically related to field validation. @pmorie is the ErrorList something we can use?